tests/cypress/e2e/unit_tests/p1_2_fleet.spec.ts

```diff
@@ -1064,6 +1064,25 @@ describe('Test move cluster to newly created workspace and deploy application to
   )
 });
 
+// Test to verify that HTML content from Helm Op's error messages is removed
```

Comment:
Just a thought, if describe is covering what's inside test, then I don't think this comment is required.

Reply:
Agree. Will remove